api/v1alpha1/clusterenvironment_types.go

```diff
@@ -35,10 +35,10 @@ type ClusterEnvironmentSpec struct {
 	Labels []string `json:"labels,omitempty"`
 
 	// Namespaces in target cluster where applications are deployed
-	ApplicationNamespaces string `json:"applicationNamespaces,omitempty"`
+	ApplicationNamespaces []string `json:"applicationNamespaces,omitempty"`
 
 	// Namespaces in target cluster where services are discovered
-	ServiceNamespaces string `json:"serviceNamespaces,omitempty"`
+	ServiceNamespaces []string `json:"serviceNamespaces,omitempty"`
```

Comment:
Can we make these API changes in a separate PR and merge it fast?

Reply:
that's a good idea, I opened the following PRs:
- #38 
- #39